src/main/java/org/codehaus/groovy/vmplugin/v8/Java8.java

```diff
@@ -636,7 +637,7 @@ private Object getInvokeSpecialHandleFallback(Method method, Object receiver) {
         }
         Class<?> declaringClass = method.getDeclaringClass();
         try {
-            return getLookupConstructor().newInstance(declaringClass, -1).
```

Comment:
Just this line I am unsure about. It will be different params passed through but I am unsure of the impact.

Reply:
The Java8 build is green, so I assume it's OK to make this change.

Reply:
I was more worried about the impact on other code bases. But there might be no issue. I'll try to find a way to trigger the line in question in an adverse way in the next day or two.

Reply:
Any findings?